cds_books/config.py

```diff
@@ -142,6 +142,16 @@ def _parse_env_bool(var_name, default=None):
         'task': 'invenio_accounts.tasks.clean_session_table',
         'schedule': timedelta(minutes=60),
     },
+    "stats-process-events": {
```

Comment:
I think we should change this: instead of re-defining CELERY_BEAT_SCHEDULE, we should extend it with the cds books config.